copyshop.py

```diff
@@ -1,20 +1,45 @@
 from string import Template
 from requests.auth import HTTPBasicAuth
 from urllib.parse import urlparse
+from jinja2 import Template
```

Comment:
We need to add this new dependency to:
https://github.com/rgauthier-sauce/copy-shop/blob/RDC/requirements.txt
https://github.com/rgauthier-sauce/copy-shop/blob/RDC/setup.py#L32

It would look something like:
```
Jinja2==2.10.3
```
and
```
install_requires=['requests','jinja2'],
```